logstash-core/logstash-core.gemspec

```diff
@@ -47,17 +47,17 @@ Gem::Specification.new do |gem|
 
   gem.platform = "java"
 
-  gem.add_runtime_dependency "pry", "~> 0.10.1"  #(Ruby license)
+  gem.add_runtime_dependency "pry", "~> 0.12.2"  #(Ruby license)
```

Comment:
pry as a runtime dependency is a bit surprising to me; are we sure it is needed at runtime and not just in test/development?

Reply:
pry in runtime is useful because we support `bin/logstash -i irb` and `bin/logstash -i pry`